UefiPayloadPkg/UefiPayloadEntry/UefiPayloadEntry.c

```diff
@@ -432,6 +432,9 @@ _ModuleEntryPoint (
     UniversalSerialPort->RegisterBase    = SerialPortInfo.BaseAddr;
     UniversalSerialPort->BaudRate        = SerialPortInfo.Baud;
     UniversalSerialPort->RegisterStride  = (UINT8)SerialPortInfo.RegWidth;
+    if (UniversalSerialPort->Header.Revision >= 2) {
+      UniversalSerialPort->ClockRate     = SerialPortInfo.InputHertz;

```

Comment:
ggua, gdong1 and myself had an offline discussion and we would like to eliminate spec update and see if we could directly set PCD here? By this way no backward compatibility concern and no spec update needed. Could you test it and let us know if this is ok?

for example:
When ParseSerialInfo() returns EFI_SUCCESS, set PCD directly with SerialPortInfo we got:
    Status = PcdSet32S (PcdSerialClockRate, SerialPortInfo->InputHertz);
    if (RETURN_ERROR (Status)) {
      return Status;
    }

Reply:
I can certainly give that a try, it would just make setting that one value inconsistent with the rest of the serial port values, which isn't ideal either.

Reply:
thanks!